app/templates/kanban.html

```diff
@@ -52,42 +52,26 @@
   
       if (!response.ok) throw new Error('Network response was not ok')
       const tasks = await response.json()
-      const todoListElement = document.getElementById('Todo')
-      tasks['Todo'].forEach(function (task) {
-        const listLink = document.createElement('a')
-        listLink.id = task.id
-        listLink.textContent = task.name
-        listLink.draggable = 'true'
-        listLink.setAttribute('ondragstart', 'drag(event)')
-        listLink.className = 'block rounded-lg my-2 p-1 hover:bg-neutral-500 bg-neutral-600 '
-        listLink.setAttribute('priority', task.priority)
-        todoListElement.appendChild(listLink)
-      })
-
-      const inProgListElement = document.getElementById('In-Progress')
-      tasks['In-Progress'].forEach(function (task) {
-        const listLink = document.createElement('a')
-        listLink.id = task.id
-
-        listLink.textContent = task.name
-        listLink.draggable = 'true'
-        listLink.setAttribute('ondragstart', 'drag(event)')
-        listLink.className = 'block rounded-lg my-2 p-1 hover:bg-neutral-500 bg-neutral-600 '
-        listLink.setAttribute('priority', task.priority)
-        inProgListElement.appendChild(listLink)
-      })
-
-      const doneListElement = document.getElementById('Done')
-      tasks['Done'].forEach(function (task) {
-        const listLink = document.createElement('a')
-        listLink.id = task.id
-        listLink.textContent = task.name
-        listLink.draggable = 'true'
-        listLink.setAttribute('ondragstart', 'drag(event)')
-        listLink.className = 'block rounded-lg my-2 p-1 hover:bg-neutral-500 bg-neutral-600 '
-        listLink.setAttribute('priority', task.priority)
-        doneListElement.appendChild(listLink)
-      })
+
+
+      for (status of ['Todo', 'In-Progress', 'Done']) {
```

Comment:
Is this extensible for when new statuses are added?

Reply:
It should be. Instead of the "hardcoded" status list, you would give it the list of custom statuses.